misc/ide.talon

```diff
@@ -3,12 +3,21 @@ app: /jetbrains/
 app: IntelliJ IDEA
 app: PyCharm
 app: PyCharm64.exe
+app: Visual Studio Code
+app: Code - OSS
+app: Code
+app: Code.exe
+app: Atom
```

Comment:
I don't think we have any Atom support yet, so I'd probably leave that out for the moment.